src/main/java/eu/darkbot/api/managers/HeroManager.java

```diff
@@ -15,20 +15,24 @@
  */
 public interface HeroManager extends Ship {
 
-    Drive getDrive();
-
     boolean hasTarget();
     Ship getTarget();
     void setTarget(Ship target);
 
-    boolean jumpPortal(Portal portal);
-
     default long timeTo(double distance) {
         return (long) (distance * 1000 / getSpeed());
     }
 
-    default long timeTo(LocationInfo location) {
-        return timeTo(getLocationInfo().distance(location));
+    default long timeTo(Location destination) {
+        return timeTo(getLocationInfo().distance(destination));
+    }
+
+    default long timeTo(LocationInfo destination) {
+        return timeTo(getLocationInfo().distance(destination));
+    }
+
+    default long timeTo(Entity destination) {
+        return timeTo(getLocationInfo().distance(destination));
     }
```

Comment:
Consider adding a generic Locatable interface with a "getLocation" method, Entity, LocationInfo and Location all implement it, and you stop having the need for 3 diff methods for this

Reply:
<:thumbsup~1:670684720639770635>